assets/src/components/notificationDrawer.tsx

```diff
@@ -130,37 +87,38 @@ const NewNotifications = ({
   )
 }
 
-const ReadNotifications = ({
-  notifications,
-  currentTime,
-  openVPPForCurrentVehicle,
-}: {
-  notifications: Notification[]
-  currentTime: Date
-  openVPPForCurrentVehicle: (notification: Notification) => void
-}) => (
-  <div className="m-notification-drawer__read-section">
-    <div className="m-notification-drawer__unread-section-header">Read</div>
-    {notifications.map((notification) => (
-      <NotificationCard
-        key={notification.id}
-        notification={notification}
-        currentTime={currentTime}
-        openVPPForCurrentVehicle={openVPPForCurrentVehicle}
-      />
-    ))}
-  </div>
+const EmptyMessage = () => (
+  <>
+    <p>You have no notifications currently.</p>
+    <p>
+      Here you'll be notified about events like accidents and ghost vehicles on
+      any bus routes you have selected on the Route Ladders page.
+    </p>
+  </>
 )
 
-const EllipsisSubmenu = ({ notification }: { notification: Notification }) => {
+const EllipsisSubmenu = ({
+  notification,
+  setShowSubmenu,
+}: {
+  notification: Notification
+  setShowSubmenu: Dispatch<SetStateAction<boolean>>
+}) => {
   const { dispatch } = useContext(NotificationsContext)
   const otherReadState = otherNotificationReadState(notification.state)
   return (
-    <div className="m-notification-drawer__submenu">
+    <div
+      className="m-notification-drawer__submenu"
+      onClick={
+        /* istanbul ignore next */
+        (event) => event.stopPropagation()
+      }
```

Comment:
My understanding from reading this is that it's intended to prevent a click in the area outside of the `<a>` but inside of the `<div>` from propagating and firing the main `onClick` event for the card, correct? How hard would it be to add a test for this? You'd basically have to pick out the right `<div>`, fire the `onClick` event, and assert that it opens the submenu and doesn't do any of the other stuff normally associated with clicking on the card.

Reply:
probably not so hard, I suspect that at the moment I wrote that `istanbul ignore next` it was late and I was tired of Codecov's bullshit. I'll give a proper test a try.